db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/SpecialFunctionNode.java

```diff
@@ -170,7 +178,7 @@ public ValueNode bindExpression(FromList fromList,
 			dtd = DataTypeDescriptor.getBuiltInDataTypeDescriptor(
 				// size: 2+(2*128) start and end text quote plus max # of
```

Comment:
this size is much larger, but the comment is unchanged?

Reply:
It's VARCHAR, so this is a max length, maybe to accommodate really long names.

Reply:
current_role could return names of multiple active roles, the original size is too small. So I just increased it to max.